.github/workflows/vcpkg.yml

```diff
@@ -207,6 +207,7 @@ jobs:
           name: build-logs-${{ matrix.triplet }}
           path: |
             ${{ env.CMAKE_BUILD_DIR }}/vcpkg-manifest-install.log
+            ${{ github.workspace }}/vcpkg/base/buildtrees/**/*.log
```

Comment:
Thanks @suricactus

Reply:
No idea how I helped, but you are welcome :)

Reply:
You mentioned that there are warnings on those builds when it tries to upload logs